docs/api.rst

```diff
@@ -0,0 +1,3 @@
+API Reference
```

Comment:
I think we can just remove this entry entirely, right? There's no API to document.